epub33/core/index.html

```diff
@@ -1863,61 +1857,61 @@ <h4>URLs in the OCF Abstract Container</h4>
 								<code>file:///path/to.epub#path=/</code>, or <code>jar:file:/path/to.epub!/EPUB/</code>
 							do not (parsing the URL string "<code>..</code>" with these three examples as base would
 							return <code>https://localhost:12345/path/</code>, <code>file:///path/</code>, and a parsing
-							error, respectively). It is the responsibility of the Reading System to assign a URL to the
+							error, respectively). It is the responsibility of the reading system to assign a URL to the
 							root directory that complies with the properties defined above. </p>
 					</div>
 
 					<div class="note">
 						<p>
-							<a data-lt="url parser">Parsing</a> may replace some characters in the File
-							Path by their <a data-cite="url#percent-encode">percent encoded</a> alternative. For
-							example, <code>A/B/C/file&#160;name.xhtml</code> becomes
-								<code>A/B/C/file%20name.xhtml</code>. </p>
+							<a data-lt="url parser">Parsing</a> may replace some characters in the file path by their <a
+								data-cite="url#percent-encode">percent encoded</a> alternative. For example,
+								<code>A/B/C/file&#160;name.xhtml</code> becomes <code>A/B/C/file%20name.xhtml</code>.
+						</p>
 					</div>
 
-					<p> A string <var>url</var> is a 
-						<dfn class="export" id="dfn-valid-relative-container-url-with-fragment-string">valid-relative-ocf-URL-with-fragment string</dfn> 
-						if it is a <a>path-relative-scheme-less-url string</a>, optionally followed by <code>U+0023 (#)</code> and
-						a <a>url-fragment string</a>, and if the following steps return <var>true</var>: </p>
+					<p> A string <var>url</var> is a <dfn class="export"
+							id="dfn-valid-relative-container-url-with-fragment-string"
+							>valid-relative-ocf-URL-with-fragment string</dfn> if it is a
```

Comment:
I wonder whether we should not use `url` instead of `URL` in this term definition. All the similar terms of the URL specification use lower case url in their definition (like "path-relative-scheme-less-url string"). The counterargument may be that we use `URL` in all our terms related to urls, like "container root URL".

We could also decide to systematically use lower case url for our other terms as well.

Reply:
> All the similar terms of the URL specification use lower case url in their definition

Only in their IDs, which we do, too. The names themselves keep the uppercase form. For example:

- [path-relative-scheme-less-URL string](https://url.spec.whatwg.org/#path-relative-scheme-less-url-string)
- [path-absolute-URL string](https://url.spec.whatwg.org/#path-absolute-url-string)